src/docs/api/public.mdx

```diff
@@ -67,36 +67,70 @@ If your answers are Yes and No, you're in business - make the endpoint public. H
+  - Response Schema
+  - Examples.
+ 
+Below we'll describe how to document these for an endpoint. You can familiarize yourself with some examples on the API docs website. We'll be making heavy use of `extend_schema` in order to do so.
+
+1. Declare the endpoint public using the `@declare_public` decorator, specifying the methods on the view which should be public.  
+```
+from sentry.apidocs.decorators import declare_public
+@declare_public(methods={"GET"})
+class OrganizationSCIMMemberDetails(...):
+```
+2. Tag the endpoint view if needed (you can also tag parent classes), this is how the endpoint makes it into a particular sidebar grouping.
+```
+from drf_spectacular.utils import extend_schema
+@extend_schema(tags=["SCIM"])
+class SCIMEndpoint(OrganizationEndpoint)...
+```
+4. Examples can be directly inserted using `OpenApiExample`, and multiple can be provided.
+5. Specify an `operation_id` (this is what is shown as the title under the sidebar).
+   - You can see the current list of tags or add tags [here](https://github.com/getsentry/sentry/blob/bc0c4f23104cc63f632fa3f6773451545720f2ec/src/sentry/apidocs/build.py#L15).
+7. Specify the parameters using a list, and a serializer if needed.
+
+   - DRF serializers work very well for query parameters, try to use them where possible.
+
+6. Specify the request and response serializers
+   - Request serializers, since they use DRF serializers, can generally just be the serializer itself. There is plenty of customization and if needed can be overriden with `inline_serializer`. These will be passed to the `request` parameter in `extend_schema`.
+   - For our response serializers, you must type the `serialize` function's return with a `TypedDict` and pass the serializer as a parameter in `extend_schema`.
+   - If the response is a wrapper or you need more customization, you can use the `inline_sentry_response_serializer` function.
+   - Note that you can instead provide OpenAPI JSON, so you can fall back to this if you are running into issues.
+7. See [here](https://github.com/getsentry/sentry/blob/master/src/sentry/scim/endpoints/members.py) for an example of an endpoint documented using drf-spectacular, and the [drf-spectacular docs here](https://drf-spectacular.readthedocs.io/en/latest/) to learn more.
 
-1. Create a new file for your schema under the appropriate folder [here](https://github.com/getsentry/sentry/tree/master/api-docs/paths). Document the schema for the relevant requests.
-2. Add the new path to [this file](https://github.com/getsentry/sentry/blob/master/api-docs/openapi.json).
-3. Write a test [here](https://github.com/getsentry/sentry/tree/master/tests/apidocs).
+**Note that if the endpoint you're modifying previously had JSON documentation, you will need to delete the old documentation found in [this file](https://github.com/getsentry/sentry/blob/master/api-docs/openapi.json)**
 
+8. Write a test [here](https://github.com/getsentry/sentry/tree/master/tests/apidocs).
```

Comment:
With sublists and code blocks I would approach this as a set of paragraphs instead of a numbered list.